storage/client_test.go

```diff
@@ -1120,6 +1120,66 @@ func TestHMACKeyCRUDEmulated(t *testing.T) {
 	})
 }
 
+func TestBucketConditionsEmulated(t *testing.T) {
+	transportClientTest(t, func(t *testing.T, project, bucket string, client storageClient) {
+		ctx := context.Background()
+		cases := []struct {
+			name string
+			call func() error
+		}{
+			{
+				name: "get",
+				call: func() error {
+					bucket, metaGen, err := createBucket(ctx, project)
+					if err != nil {
+						return fmt.Errorf("creating bucket: %w", err)
+					}
```

Comment:
Can this be refactored out into t.Run and passed into the call?

Reply:
sure, done